web/src/app/rotations/RotationForm.js

```diff
@@ -111,16 +108,49 @@ export default function RotationForm(props) {
             max={9000}
           />
         </Grid>
-        <Grid item xs={12}>
+        {localZone !== props.value.timeZone && (
+          <Grid item xs={12}>
+            <TimeZoneSwitch option={props.value.timeZone} />
+          </Grid>
+        )}
+        <Grid item xs={6}>
           <FormField
             fullWidth
-            component={ISOTimePicker}
-            label='Handoff Time'
+            component={ISODateTimePicker}
+            label='Initial Handoff Time'
             name='start'
             required
           />
         </Grid>
-        {props.value.type === 'weekly' && renderDayOfWeekField()}
+        <Grid item xs={6}>
+          <List
+            dense
+            disablePadding
+            subheader={
+              <ListItem disableGutters className={classes.noVerticalSpace}>
+                <ListItemText
+                  primary='Upcoming Handoff Times:'
+                  primaryTypographyProps={{ className: classes.semiBold }}
+                />
+              </ListItem>
+            }
+          >
+            {nextHandoffs.map((text, i) => {
+              return (
+                <ListItem
+                  key={i}
+                  className={classes.noVerticalSpace}
+                  disableGutters
+                >
+                  <ListItemText
+                    primary={text}
+                    className={classes.noVerticalSpace}
+                  />
+                </ListItem>
+              )
+            })}
+          </List>
```

Comment:
Since you are creating a MUI list here but then removing the padding and such with:
- `dense`
- `disablePadding`
- `disableGutters`
- `className={classes.noVerticalSpace}`

We can just render as `Typography` as this will evaluate to multiple `p` tags under a `div`, which will stack as intended without padding, and removes the dependency on `makeStyles`.

```suggestion
          <Typography>
            <strong>Upcoming Handoff Times:</strong>
          </Typography>
          {nextHandoffs.map((text, i) => (
            <Typography key={i}>{text}</Typography>
          ))}
```